templates/antora/antora.converter.js

```diff
@@ -0,0 +1,334 @@
+module.exports = function () {
+
+  // Todo we can pass this through later
+  var baseurl = '';
+
+  function hasValue (x) {
+    // empty helper for strings, objects, arrays
+    if (typeof(x) === 'string' || typeof(x) === 'array') {
+      return x.length > 0
+    } else if (typeof(x) === 'object') {
+      return Object.keys(x).length > 0
+    } else {
+      return false;
+    }
+  };
+  // uppercase first char
+  function UCFirst (str) {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+  }
+  // runs a bunch of required cleanup filters, where embedded code/text can break asciidoc rendering
+  function cleanFilter (str) {
```

Comment:
our source code comments can contain HTML descriptions/examples/links which can break asciidoc syntax, when that happens we add a filter to escape chars etc as required.

Reply:
It mostly works quite well and makes it far more simple, i just gave it a try and there are some cases where it does get fuzzed up (probably not hard to fix)
![Screen Shot 2022-02-16 at 3 40 47 pm](https://user-images.githubusercontent.com/39354587/154203400-70c48001-7eaa-43ec-856b-5ed0320cee8d.png)
)

We would still need to add filters to cleanup some of the broken bits and also convert back tick ` to <em>
For now I will just leave it as is